src/Symfony/Component/HttpFoundation/ParameterBag.php

```diff
@@ -188,6 +188,35 @@ public function getBoolean($key, $default = false)
     }
 
     /**
+     * Returns the parameter value converted to a DateTime object.
+     *
+     * @param string             $key      The parameter key
+     * @param string             $format   The expected date format
+     * @param string             $default  The default value to be converted to a DateTime object if the parameter key does not exist
```

Comment:
@xabbuh I stick to my first pov: all the other getter (getAlpha, etc.) start with something like `$this->get($key, $default)` then do transform the result.